infra/terraform/external_function/snowalert/cloudwatch.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_iam_policy" "prod_cloudwatch_write" {
-  name = "cloudwatch-setup-and-write"
+  name = var.prod_cloudwatch_write_name
```

Comment:
generally, users do not care what the policies are called, so no need to have it as a variable